lib/email-validation.rb

```diff
@@ -3,8 +3,12 @@
 require 'email-validation/config'
 
 module EmailValidation
-  def self.incorrect_email_message(email)
-    "The email address you have entered (#{email}) is incorrect."
+  def self.invalid_email_message
+    "Your email address is invalid. Please #{self.contact_link} if you need further assistance."
```

Comment:
My concern is that the contact link will be escaped through Rails. This leads me to thought that both messages should be configurable and passed to the gem with `.html_safe` from Rails